### DISCUSSION TITLE
Welcome to Grounded-SAM-2 Discussions!

### DISCUSSION THREAD

**Discussion by rentainhe** [IDEA-Research/Grounded-SAM-2#27] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EmbodiedMind** [comment]
Hi authors! I'm currently working on real-time visual tracking and inference based on this project. I've successfully deployed the model on a camera stream and would love to share my implementation with the community.

Would it be okay for me to upload the real-time inference and tracking code based on this project?

**Reply by EmbodiedMind** [reply]
![实时追踪 -middle-original](https://github.com/user-attachments/assets/90c41344-bdd3-4fa5-8be1-097c56d15577)

**Reply by EmbodiedMind** [reply]
The images on the bottom left and right were generated through real-time inference on the 4090 GPU. The top left and bottom left images demonstrate the signal transmission latency between the master and slave nodes in the ROS system.
